### DISCUSSION TITLE
Unable to delete a trigger created in the wrong table

### DISCUSSION THREAD

**Discussion by srowe0091** [supabase/supabase#16975] [Contribute to Supabase]
I have a feeling this might be a bug with permissions. I accidentally created a trigger in the wrong table, the `_http_response` was selected by default when I was testing a trigger, but once I realized where that trigger was created, I went to delete it, but I can't delete it because im getting this error `ERROR: must be owner of relation _http_response`

How was I able to add a trigger to the table, but not be able to delete it? 

It doesn't appear in the dashboard, and I can't delete it manually with SQL editor.

How can I delete this trigger?

**Comment by Hallidayo** [comment]
Hi @srowe0091, Does the trigger appear in the Database > Trigger section on the dashboard?

**Comment by srowe0091** [comment]
@Hallidayo no it does not. Support was able to get this resolved by having them remove the trigger for me, but I guess this issue can stay open because I was able to do something I shouldn't have?

**Comment by mancvso** [comment]
HI

I can confirm the same issue.
Trigger does not appear and deleting it from SQL results in Permission denied for table pg_trigger.

**Comment by bentron2000** [comment]
I also had this issue: Was able to create but not delete with that same error.
Can supabase confirm if adding triggers to this table is possible?
